frameworks/helloworld/src/test/java/com/mesosphere/sdk/helloworld/scheduler/HelloWorldServiceSpecTest.java

```diff
@@ -41,9 +43,28 @@ public void beforeEach() {
     }
 
     @Test
-    public void testServiceSpecDeserialization() throws Exception {
-        ClassLoader classLoader = HelloWorldServiceSpecTest.class.getClassLoader();
-        File file = new File(classLoader.getResource("svc.yml").getFile());
+    public void testDefaultDeserialization() throws Exception {
+        testDeserialization("svc.yml");
+    }
+
+    @Test
+    public void testDefaultValidation() throws Exception {
+        testValidation("svc.yml");
+    }
+
+    @Test
+    public void testOneTimePlanDeserialization() throws Exception {
+        testDeserialization("svc_plan.yml");
```

Comment:
Not sure.  I've run across, a couple validation things that aren't as one would expect.  I'll file a JIRA to verify/fix.